README.md

```diff
@@ -43,14 +43,14 @@ framework are exposed.
 ## Repository Management
 
 CodeIgniter is developed completely on a volunteer basis. As such, please give up to 7 days
-for your issues to be reviewed. If you haven't heard from one of the team in that time period,
+for your issues to be reviewed. If you haven't heard from one of the teams in that time period,
```

Comment:
Please revert. `team` here is used to refer to the collective so plural form is not desired here.